src/txHashBuilder_test.c

```diff
@@ -371,7 +371,9 @@ void run_txHashBuilder_test()
 	                   true, // metadata
 	                   true, // validity interval start
 	                   true, // mint
-	                   false // script hash data
+	                   false, // script hash data
+	                   0,	//TODO(KoMa) testing
+	                   0	//TODO(KoMa) testing
```

Comment:
What about this?

Reply:
I already disabled the self tests, for the lack of space on the device, I'm not sure there's much point to improve on them at this point.

Reply:
I'm OK with not improving them further, but please make the comment more detailed (e.g. "collaterals not used in this basic test" or something). That would allow to remove the TODO.